CONTRIBUTING.md

```diff
@@ -1,3 +1,10 @@
 # Contributing
 
-Please have a look at our [documentation site](https://subnauticanitrox.github.io/Documentation/) for contribution guidelines & information on how to get started.
\ No newline at end of file
+Please have a look at our [documentation site](https://subnauticanitrox.github.io/Documentation/) for contribution guidelines & information on how to get started.
+
+## Stand on AI and generated issues & prs
+
+We generally don't want to tell you what tools to use (you undoubting minimalistic vi enjoyer), but we ideally want to have clean and minimal code, issues and pull requests. \
+For this reason we are ceasing the creation of AI generated issue and pr descriptions. We don't have the resources to read 50 paragraphs on why changing 5 lines of code makes sense. Only explain yourself if the goal your implementation is not a no-brainer. \
+You are free to use AI to generate your code but we require you to review it before submitting. Nitrox is a special kind of software as it's interacting tightly with Subnautica's code base which makes good code generation difficult. Also keep in mind Subnautica's source code is under copyright and it's prohibited to upload it to an AI. \
+As a last note, only place comments on functions which meaning is not obvious. Follow the principle of "Good codes don't need comment". A good example for where comments are needed/expected are the Harmony patches.
```

Comment:
I think that this line is probably more generic than the AI specific. I do agree with the idea behind it though, maybe under a code standards bit?